community/kernel/src/main/java/org/neo4j/kernel/impl/store/CommonAbstractStore.java

```diff
@@ -851,18 +851,6 @@ protected void assertNotClosed()
     @Override
     public void close()
     {
-        if ( idGenerator == null || !storeOk )
-        {
-            try
-            {
-                closeStoreFile();
```

Comment:
I see no reason to not remove stupid code in an old branch. Being super religious about this stuff is not very helpful.